src/logaze/transform.clj

```diff
@@ -46,32 +47,57 @@
   (when-let [match (re-find #"\d+ ?GB" memory-str)]
     (string/replace match " " "")))
 
-(defn resolution
-  [display-str]
-  (when-let [match (re-find #"\d+\s*x\s*\d+" display-str)]
-    (string/replace match " " "")))
+(defn match-regex-and-return [str regex & {:keys [result]}]
+  (let [value (re-find regex str)]
+    (if (and (some? value) (some? result))
+      result
+      value)))
+
+(defn get-resolution
```

Comment:
Using a noun without the `get-` prefix is a clojure thing: https://stuartsierra.com/2016/01/09/how-to-name-clojure-functions

Reply:
Oh that's neat, I didn't realize. Initially I was going to keep the original `resolution` method, I think that's why I came up with `get-resolution` Interesting style for Clojure, I like it! Will change this.